__tests__/installer.test.ts

```diff
@@ -55,22 +55,33 @@ describe('version matching tests', () => {
             expect(installer.getJuliaVersion([], 'nightly')).toEqual('nightly')
             expect(installer.getJuliaVersion(testVersions, 'nightly')).toEqual('nightly')
         })
+
+        it('LTS', () => {
+            // Update test when LTS is updated
+            expect(installer.getJuliaVersion(testVersions, 'lts')).toEqual(installer.getJuliaVersion(testVersions, '1.6'))
+            expect(installer.getJuliaVersion(testVersions, 'lts')).toEqual('1.6.7')
+        })
+
+        it('pre', () => {
+            expect(installer.getJuliaVersion(testVersions, 'pre')).toEqual('1.11.0-beta1')
```

Comment:
@KristofferC Will this test will need to be updated every time there's a new pre-release?